Makefile

```
@@ -24,7 +24,7 @@ GOBUILDFLAGS = -gcflags="all=-trimpath=$(GOPATH)" -asmflags="all=-trimpath=$(GOP
 SELECTOR_SYNC_SET_HOOK_EXCLUDES ?= debug-hook
 SELECTOR_SYNC_SET_DESTINATION = build/selectorsyncset.yaml
 
-CONTAINER_ENGINE ?= docker
+CONTAINER_ENGINE=$(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
```

```suggestion
CONTAINER_ENGINE = $(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
```

The most minor of nits.

---

And why this instead of overriding at runtime as in `make CONTAINER_ENGINE=podman`?

---

this is, by far, the better UX